testing/test_functions.py

```diff
@@ -34,11 +34,11 @@ def test_next_tag(tag, expected):
 
 
 VERSIONS = {
-    "exact": meta("1.1", None, False),
-    "zerodistance": meta("1.1", 0, False),
-    "dirty": meta("1.1", None, True),
-    "distance": meta("1.1", 3, False),
-    "distancedirty": meta("1.1", 3, True),
+    "exact": meta("1.1", distance=None, dirty=False),
```

Comment:
this change here really makes things more beautiful,
i'd like to keep it even tho it was made necessary by the accidental api break before